### DISCUSSION TITLE
Remotion Player with long videos

### DISCUSSION THREAD

**Discussion by andreidimaano** [remotion-dev/remotion#1896] [Q&A]
hello :) I want to use remotion player on long mp4 videos - 3 to 5 minutes long. Currently, I'm using HLS for video streaming but I'm considering using the Player for rendering Text based on user input. How is the performance of remotion player with longer videos?

**Comment by JonnyBurger** [comment]
Hey Andrei!

Thanks for the question.
The Remotion Player actually only renders the current frame at any time. So it does not matter how long the video is, the performance should be the same!

(This of course can also be a disadvantage if the rendering is slower than real-time, or if you need to preload a video, but this is the tradeoff that we are taking.)
